pkg/controller/workload/bpfcache/service.go

```diff
@@ -32,9 +32,9 @@ type ServicePorts [MaxPortNum]uint32
 type TargetPorts [MaxPortNum]uint32
 
 type ServiceValue struct {
-	EndpointCount uint32       // endpoint count of current service
-	LbPolicy      uint32       // load balancing algorithm, currently only supports random algorithm
-	ServicePort   ServicePorts // ServicePort[i] and TargetPort[i] are a pair, i starts from 0 and max value is MaxPortNum-1
+	EndpointCount [PrioCount]uint32 // endpoint count of current service
```

Comment:
map

Reply:
If using map here, how to update this value to bpf map? maintain a new bpf map stuct?